### PR TITLE
docs(obs): fix docs issues

### PR DESCRIPTION
**Pull request by Zippo-Wang**
fix docs issues

### CODE REVIEW

File: flexibleengine/resource_flexibleengine_s3_bucket.go [hunk 1]
```diff
@@ -140,23 +139,21 @@ func resourceS3Bucket() *schema.Resource {
...
 
 			"region": {
 				Type:     schema.TypeString,
 				Optional: true,
+				ForceNew: true,
 				Computed: true,
 			},
 			"website_endpoint": {
 				Type:     schema.TypeString,
-				Optional: true,
```
Comment (ShiChangkuo) [inline flexibleengine/resource_flexibleengine_s3_bucket.go:154]: it's better to put all `computed` attrs at the end of schema.

Comment (Zippo-Wang) [reply flexibleengine/resource_flexibleengine_s3_bucket.go:154]: I have fix the problem, thanks for your advice

File: docs/resources/s3_bucket.md [hunk 2]
```diff
@@ -142,93 +142,139 @@ resource "flexibleengine_s3_bucket" "versioning_bucket" {
...
+* `enabled` - (Required, Bool) Specifies lifecycle rule status.
+
+* `abort_incomplete_multipart_upload_days` (Optional, Int) Specifies the number of days after initiating a multipart
+  upload when the multipart upload must be completed.
+
+* `expiration` - (Optional, List) Specifies a period in the object's expire.
+  The [expiration](#obs_expiration) object structure is documented below.
+
+* `noncurrent_version_expiration` - (Optional, List) Specifies when noncurrent object versions expire.
+  The [noncurrent_version_expiration](#obs_noncurrent_version_expiration) object structure is documented below.
```
[the remark this answers is not in the document]
Comment (Zippo-Wang) [reply docs/resources/s3_bucket.md:242]: this field is right

**Pull request merged**